app/src/main/java/com/blackboxteam/app/blackbox/LaunchScreenActivity.java

```diff
@@ -37,4 +43,16 @@ public void launchSignupScreenActivity (){
     protected void attachBaseContext (Context newBase) {
         super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase));
     }
+
```

Comment:
Once you are done implementing the UserSessionPersistence class, remove all method calls in activity.